### PR TITLE
Enable watchdog

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board [hunk 1]
```diff
@@ -57,6 +57,11 @@ sync
 
 [ -z $INSTALL_UEFI ] && exit 0
 
+# remove existing resinOS entries so we won't have duplicates
```
Comment (agherzan) [inline layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: balenaOS

Comment (floion) [reply layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: The entry is still named resinOS currently.

**Comment by floion** [comment]
@balena-ci retest

File: layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board (hunk 1 above)
Comment (agherzan) [reply layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: I see.

**Pull request merged**